django/core/management/helper.py

```diff
@@ -0,0 +1,66 @@
+from django import setup
+from django.apps import apps
+from django.utils.module_loading import import_string
+
+DJANGO_DEFAULT_IMPORTS = [
+    "from django.core.cache import cache",
+    "from django.conf import settings",
+    "from django.contrib.auth import get_user_model",
+    "from django.db import transaction",
+    "from django.db.models import Avg, Case, Count, F, Max, Min, Prefetch, Q, Sum, When",
+    "from django.utils import timezone",
+    "from django.urls import reverse",
+    "from django.db.models import Exists, OuterRef, Subquery",
+]
+
+CHANGED_IMPORTS = [
+    {"cache": "django.core.cache.cache"},
+    {"settings": "django.conf.settings"},
+    {"get_user_model": "django.contrib.auth.get_user_model"},
+    {"transaction": "django.db.transaction"},
+    {"Avg": "django.db.models.Avg"},
+    {"Case": "django.db.models.Case"},
+    {"Count": "django.db.models.Count"},
+    {"F": "django.db.models.F"},
+    {"Max": "django.db.models.Max"},
+    {"Min": "django.db.models.Min"},
+    {"Prefetch": "django.db.models.Prefetch"},
+    {"Q": "django.db.models.Q"},
+    {"Sum": "django.db.models.Sum"},
+    {"When": "django.db.models.When"},
+    {"timezone": "django.utils.timezone"},
+    {"reverse": "django.urls.reverse"},
+    {"Exists": "django.db.models.Exists"},
+]
+
+
+def get_objects(style):
+    import_objects = {}
+    for x in CHANGED_IMPORTS:
+        for key, val in x.items():
+            import_objects[key] = import_string(val)
+    print(style.MIGRATE_HEADING("# Importing Necessary modules"))
+    for x in DJANGO_DEFAULT_IMPORTS:
+        print(style.SUCCESS(x))
```

Comment:
As discussed on the forum, I’d prefer we only write a single message like: “Imported N default objects.”. Although there’s probably better wording that doesn’t sound like a data import.

Reply:
How about this approach ?
![Screenshot from 2024-02-07 11-45-46](https://github.com/django/django/assets/93030127/a2b14193-f669-4d24-84a6-e8c3869a1db9)